python.en.md

```diff
@@ -0,0 +1,91 @@
+# Python coding style
+
+## Introduction
+
+This guide defines the conventions for writing Python code at Cookpad Inc.
+
+## General rules:
+
+ - value readability and maintainability
+ - keep it simple
+ - be consistent in your style 
+ - for things not in this guide, be consistent with the style of the project you're working on
+
+## Python version
+
+ - **[SHOULD]** When possible, projects should use Python 3 syntax. Use of Python 2 should be motivated by external reasons, like dependencies or heavy use of legacy code.
+
+## Language Features
+
+🙌Avoid:
+
+ - global variables
+
+👋Use for simple cases (one line):
+
+ - lambda functions
+ - list comprehensions `[ingredient.name for ingredients in recipe]`
+ - conditional expressions `sugar = 1 if 'milk' in coffee else 0`
+ - nested/local/inner classes and functions
+
+👍Use freely:
+
+ - generators
+ - default iterators and operators for types that support them
+
+## Code style
+
+### Lint
+
+Run `pep8` on your code.
+
+### Comments and docstrings
+
+Comment and document your code for others to understand. Use best judgement when commenting: if anything may be hard to understand for someone reading the code for the first time, add a comment. 
+
+ - **[MUST]** Use simple, easy to understand English
+ - **[SHOULD]** Use docstrings for modules
+ - **[SHOULD]** Use docstrings for classes and methods that are accessible outside your module
+
+### Indentation
+
+Indent code blocks with 4 spaces.
+
+### White space
+
+ - **[MUST]** Spaces before and after operators, except in keyword arguments or parameter values.
+ - **[MUST]** Single space after comma, no space before
+
+```python
+def sample_function(x, y=0):
+    y = y + x
+    if x == 4:
+        print(x, y)
+```
+
+ - **[SHOULD]** Two blank lines between top level definitions, one blank line between method definitions
+ - **[MUST]** Avoid ending lines with semicolons
+ - **[SHOULD]** Keep line length up to 80 characters
```

Comment:
FYI I think this is antiquated. I personally do kind of prefer to have lines shorter than, say, 80 or at least 120, but it can be counterproductive too.